osvr_tracked_device.h

```diff
@@ -488,7 +455,16 @@ const char* OSVRTrackedDevice::GetSerialNumber()
 
 float OSVRTrackedDevice::GetIPD()
 {
-    return static_cast<float>(m_DisplayConfiguration->getIPDMeters());
+    OSVR_Pose3 leftEye, rightEye;
+    if(m_DisplayConfig.getViewer(0).getEye(0).getPose(leftEye) != true) {
+        logger_->Log("OSVRTrackedDevice::GetHeadFromEyePose(): Unable to get left eye pose!\n");
+    }
+    if(m_DisplayConfig.getViewer(0).getEye(1).getPose(rightEye) != true) {
+        logger_->Log("OSVRTrackedDevice::GetHeadFromEyePose(): Unable to get right eye pose!\n");
+    }
+    Eigen::Map<Eigen::Vector3d> lT = osvr::util::vecMap(leftEye.translation);
+    Eigen::Map<Eigen::Vector3d> rT = osvr::util::vecMap(rightEye.translation);
+    return 2*std::sqrt(std::abs(rT.dot(lT)));
```

Comment:
Why do we need this factor of 2 here to get the expected IPD value? (I'm probably just forgetting my linear algebra, it's been a while...)